<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="email">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeEmail">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Age">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Age:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LeAge">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="phoneNumber">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LePhoneNumber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>+62 000 0000 0000</string>
    </property>
    <property name="text">
     <string>+62   </string>
    </property>
   </widget>
   <widget class="QLabel" name="address">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TeAddress">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="gender">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CbGender">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="education">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Education:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CbEducation">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kindergarten</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Elementary School</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Junior High School</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Senior High School</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vocational High School</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bachelor's Degree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diploma Degree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Master's Degree</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Doctorate (Ph.D)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <zorder>education</zorder>
   <zorder>name</zorder>
   <zorder>LeName</zorder>
   <zorder>email</zorder>
   <zorder>LeEmail</zorder>
   <zorder>Age</zorder>
   <zorder>LeAge</zorder>
   <zorder>phoneNumber</zorder>
   <zorder>LePhoneNumber</zorder>
   <zorder>address</zorder>
   <zorder>TeAddress</zorder>
   <zorder>gender</zorder>
   <zorder>CbGender</zorder>
   <zorder>CbEducation</zorder>
   <zorder>btnSave</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuValidation_Form">
    <property name="title">
     <string>F1D022043_Faizah</string>
    </property>
   </widget>
   <addaction name="menuValidation_Form"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
